### PR TITLE
Merge slide-hits into master & release 1.1.0

### PR DESCRIPTION
**Pull request by paulriddle**

### CODE REVIEW

**Comment by paulriddle** [comment]
@Le6ow5k1 @Napolskih

File: lib/redis_counters/dumpers/version.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module RedisCounters
   module Dumpers
-    VERSION = '1.0.0'
+    VERSION = '1.1.0'
```
Comment (apress-hound) [inline lib/redis_counters/dumpers/version.rb]: Freeze mutable objects assigned to constants.

Comment (apress-hound) [inline lib/redis_counters/dumpers/version.rb]: Freeze mutable objects assigned to constants.

**Comment by paulriddle** [comment]
@bibendi

**Comment by paulriddle** [comment]
@Napolskih up
Кто-нибудь выпустите версию на rubygems.

**Comment by Napolskih** [comment]
@Le6ow5k1

**Pull request merged**